components/StartEndDate.vue

```diff
@@ -48,36 +46,9 @@
 <script>
 export default {
   props: {
-    uniqueid: {
-      type: String,
+    value: {
+      type: Object,
       required: true
-    },
-    start: {
-      validator: prop => typeof prop === 'string' || prop === null,
```

Comment:
I don't like these validator functions, it basically is undoing the work of `required: true` if null is allowed...